smrf/tests/data/test_gridded_input.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+from unittest.mock import MagicMock
+
+import pandas as pd
+
+from smrf.data.gridded_input import GriddedInput
+from smrf.data.load_topo import Topo
+
+
+class TestDataSource(GriddedInput):
+    pass
+
+
+class TestGriddedInput(unittest.TestCase):
+    TOPO_MOCK = MagicMock(spec=Topo, instance=True)
+    BBOX = [1, 2, 3, 4]
+    CONFIG = MagicMock('config')
+    START_DATE = pd.to_datetime('2021-01-01 00:00 UTC')
+    END_DATE = pd.to_datetime('2021-01-02')
+    VALID_ARGS = dict(
+        start_date=START_DATE, end_date=END_DATE,
+        topo=TOPO_MOCK, bbox=BBOX, config=CONFIG
+    )
+
+    def test_start_date(self):
+        hrrr_input = GriddedInput(**self.VALID_ARGS)
+
+        self.assertEqual(
+            self.START_DATE,
+            hrrr_input.start_date
+        )
+
+    def test_end_date(self):
+        hrrr_input = GriddedInput(**self.VALID_ARGS)
+
+        self.assertEqual(
+            self.END_DATE,
+            hrrr_input.end_date
+        )
+
+    def test_time_zone(self):
```

Comment:
For completeness and since you've done a lot of work with timezones, does it make sense to add a single test for a non UTC timezone? While I hope if doesn't happen, if someone is using gridded input and specifies the start date for SMRF in MST then this might have issue. Or do we enforce a check in `model_framework` that one has to run SMRF in UTC with gridded data?

Reply:
Good old timezones.

I think we are good for now. The arguments passed to the different input data classes are coming from the parsed `.ini`. If we were to enforce certain timezones, then I would vote to put that up higher in the stack. As you noted once earlier, be our guest if you want to run the model in non-UTC, but expect surprises.

This test mainly covers that the class is using the zone from the start date.

Reply:
Timezones... gotta love them in life, hate them in programming!

Probably should add something to the read the docs and the README about timezones.